### ISSUE TITLE
0.5.0 to clojars & reflection warnings

### ISSUE THREAD

**Issue by ikitommi** [venantius/ultra#64]
Hi,

`0.5.0` doesn't seem to be in Clojars yet. Built the latest myself locally, is awesome, but yields:

```clj
Reflection warning, ultra/test/diff/string.clj:26:25 - reference to field operation can't be resolved.
Reflection warning, ultra/test/diff/string.clj:28:31 - reference to field text can't be resolved.
Reflection warning, ultra/test/diff/string.clj:28:24 - reference to field trim can't be resolved.
```

**Comment by venantius** [comment]
Can you provide a project.clj and any related profile files? Also, what JDK are you on? I don't have this issue in a clean project using Ultra HEAD.

**Comment by ikitommi** [comment]
Retested against a head, and fixed it with (https://github.com/venantius/ultra/pull/65).

cheers,

Tommi

**Comment by emlyn** [comment]
What's the plan for releasing 0.5.0 to clojars, can we expect it soon?

**Comment by venantius** [comment]
Resolved by #65. Will be releasing 0.5.0 shortly.

**Comment by venantius** [comment]
0.5.0 is now live.
